phat/contracts/action_offchain_rollup/lib.rs

```diff
@@ -476,21 +495,29 @@ mod action_offchain_rollup {
         fn ensure_client_configured(&self) -> Result<Client> {
             let client = Client {
                 rpc: self.get_profile_rpc()?,
-                client_addr: self.client_addr.clone().ok_or(Error::ClientNotConfigured)?,
+                client_addr: self.client_addr.ok_or(Error::ClientNotConfigured)?,
             };
             Ok(client)
         }
 
-        fn config_core_inner(&mut self, code_hash: Hash, settings: String) -> Result<()> {
+        fn config_core_inner(&mut self, core: Core) -> Result<()> {
             get_code_provider()?
-                .use_code(code_hash)
+                .use_code(core.code_hash)
                 .or(Err(Error::CodeNotFound))?;
-            self.core.set(&Core {
-                settings,
-                code_hash,
-            });
+            self.core.set(&core);
             Ok(())
         }
```

Comment:
Can we log a warning message here when users configure to use `JsDelegate2`?

Reply:
After thinking twice, I removed the `JsDelegate2` as we won't be launching it on the mainnet. Removing it to speed up deprecating it from PoC6.
Note: `action_offchain_rollup` v2.0.1 uses `JsDelegate2`.

Reply:
Since we haven't used action_offchain_rollup yet, it is safe to remove it. Can we skip `v2.0.1` of action_offchain_rollup and move directly to the `AsyncJsRuntime` version?

Reply:
> Since we haven't used action_offchain_rollup yet, it is safe to remove it.

Good news. We should skip it.